crates/bevy_gizmos/src/lib.rs

```diff
@@ -85,15 +87,33 @@ impl Plugin for GizmoPlugin {
             .add_plugins(RenderAssetPlugin::<LineGizmo>::default())
             .init_resource::<LineGizmoHandles>()
             .init_resource::<GizmoConfig>()
-            .init_resource::<GizmoStorage>()
-            .add_systems(Last, update_gizmo_meshes)
+            .init_resource::<GizmoStorage<Swap>>()
+            .init_resource::<GizmoStorage<()>>()
```

Comment:
Looks like initially, the default buffer had the `Update` type, now it uses `()`. What's the reasoning behind changing it? I thought it was clearer.

Reply:
Well, I don't really care much either way. This one gets swapped out for other buffers, so someone looking at that might get confused now that gizmos requested in a `GizmoStorage<Update>` are cleared in `FixedMain`.

Ideally I guess it'd be that we have both `()` and `Update` but I don't think the complexity of adding that is worthwhile. Maybe if we wanted to add the ability to query `Gizmos<Update>` inside other contexts I suppose?